example/usecase/employer_university_flow/pkg/util.go

```diff
@@ -39,35 +39,34 @@ func GenerateDIDPeer() (did.DID, error) {
 	if err != nil {
 		return nil, err
 	}
-	did, err := did.PeerMethod0{}.Generate(kt, pubKey)
+	dID, err := did.PeerMethod0{}.Generate(kt, pubKey)
```

Comment:
think this should be reverted, it's either `did` or `DID`

Reply:
Using `did` would hid the package named `did`. I opted for `peer` instead, which is what the Generate method returns.